man/systemd.exec.xml

```diff
@@ -4184,6 +4184,18 @@ StandardInputData=V2XigLJyZSBubyBzdHJhbmdlcnMgdG8gbG92ZQpZb3Uga25vdyB0aGUgcnVsZX
           <xi:include href="version-info.xml" xpointer="v254"/></listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><varname>$DEBUG_STARTUP</varname></term>
```

Comment:
is that a good name? after all "startup" sounds like an initial start attempt, but this variable will only be set on starts that aren't the initial one. 

also, I think we should try to find a name that is unlikely to clash with something other people pick, given that we don't namespace these generally. 

How about DEBUG_INVOCATION=1?

Reply:
Does it have to be a boolean -- would this express the purpose
SD_SERVICE_FAILS_SINCE_LAST_CLEAN_START?

Reply:
We use the SYSTEMD_ env var prefix only for stuff that is strictly and exclusively a systemd thing. I think this here is a more generic thing, that any service manager could implement, hence i don't think we should prefix with with SYSTEMD_

(we never use the SD_ prefix for env vars)

Reply:
i'd be fine making this a counter. right now we don't keep such a counter though. would have to be added.

Reply:
> is that a good name?

I am 99% sure it's the one you suggested :-P but ok, changed to DEBUG_INVOCATION